### ISSUE TITLE
How to change the tab text color?

### ISSUE THREAD

**Issue by ghostx31** [andreasgrafen/cascade#24]
I changed the colour of the tabs to a lighter shade and I wish to change the colour of tab text (not just the selected tab but all the tabs since the background of all tabs is a bluish colour) to a darker shade but so far I have been unsuccessful. I couldn't find anything about changing tab text on google and I am lost right now. 
Can you point me on how to accomplish this?
Thanks!

**Comment by andreasgrafen** [comment]
Hey there!
I think the solution would be to set [`--lwt-text-color` Line#173](https://github.com/andreasgrafen/cascade/blob/main/userChrome.css#L173) to a colour or your liking. This will affect the inactive tabs without introducing unwanted behaviour – at least from what I can tell after some quick checks. To adjust the active tabs colour you can use [`--lwt-tab-text` Line#174](https://github.com/andreasgrafen/cascade/blob/main/userChrome.css#L174). c:

**Comment by ghostx31** [comment]
Hmm... That's strange. That's what I've been trying to do for the past two days. I've tried various stuff with it but the text always remains white. I tried changing the colour with the default cascade css but the text still remains white. It stays white even if changing to the white colour scheme when it should turn black since the inverted colour in light colour scheme is `#1E2021`. 

> at least from what I can tell after some quick checks. 
Does the colour change work for you? I am running the latest firefox on Arch Linux, if that's a needed information.

**Comment by andreasgrafen** [comment]
I've tested this on MacOS Catalina and Pop_os! 21.10 with no issues. I don't think Arch would behave any differently, but I can try it on an Arch install later too just to make sure. o:

![tabcolours](https://user-images.githubusercontent.com/35840154/152331346-c403864c-342b-4583-b02b-144e149363e5.gif)

```css
    --lwt-text-color:  hotpink !important;
    --lwt-tab-text:    blue !important;
```
<br>

The fact that it doesn't seem to change **anything** for you is really weird. How are you trying to make these changes? Are you using the remote debug for testing, or are you editing the userChrome directly in an external editor?

**Comment by ghostx31** [comment]
> How are you trying to make these changes? Are you using the remote debug for testing, or are you editing the userChrome directly in an external editor? 

I'm editing the userChrome directly in nvim. Am I not supposed to do it that way? 😅

I haven't tried simple colour names like you've done. I've used either `var()` or hex codes for colours, though I don't think that should be any problem. I'll try with simple colour names as well. I'm currently AFK but I'll let you know if it works. I'll try on another machine as well.

**Comment by andreasgrafen** [comment]
> I'm editing the userChrome directly in nvim. Am I not supposed to do it that way? sweat_smile

No, that's totally fine. **But** it doesn't update it live inside the browser when you save the file. You'd need to reload the userChrome to take effect by restarting the browser every time you make a change. I've fallen into that trap myself..
For playing around with colours and variables I usually use the [live debugging](https://www.reddit.com/r/FirefoxCSS/comments/73dvty/tutorial_how_to_create_and_livedebug_userchromecss/) capabilities of Firefox.
<br>
 
> I haven't tried simple colour names like you've done. I've used either `var()` or hex codes for colours, though I don't think that should be any problem. I'll try with simple colour names as well. I'm currently AFK but I'll let you know if it works. I'll try on another machine as well.

That shouldn't make any difference whatsoever actually. It's just something I'm quite used to at this point.. to take the most annoying colour names and throw them together for testing. :D

**Comment by ghostx31** [comment]
Okay I tested this on a Windows system as well as firefox developer on my system and it works on both of them. The developer edition is on version 97.0b9-1 while the Windows firefox is on the same version as the firefox on Arch which is 96.0.3. I am so confused right now LOL.

**Comment by andreasgrafen** [comment]
I've just tested it on Firefox 96 on Arco (which is the closest I have to baseline Arch rn) and it worked without any issues. So .. idk. This is really quite weird. o:

**Comment by ghostx31** [comment]
I'll see if reinstalling firefox does anything cause that's the only option right now. Closing this for now. 

Thank you!

**Comment by andreasgrafen** [comment]
Alright! Feel free to keep me updated on your results. c:

**Comment by ghostx31** [comment]
Just wanted to let you know that the colour change wasn't working because I was using the system theme for firefox (which was selected by default) and it doesn't use the `--lwt` variables. I didn't know the theme would be causing the issue but its solved now. Thank you :)

**Comment by andreasgrafen** [comment]
It doesn't? That is really good to know! I should look into that to figure out what variables are used then and get that fixed. Thanks a lot for letting me know! c:
